WordPress/src/main/java/org/wordpress/android/ui/posts/editor/EditorTracker.kt

```diff
@@ -109,6 +109,8 @@ class EditorTracker @Inject constructor(
             TrackableEvent.UNDERLINE_BUTTON_TAPPED -> Stat.EDITOR_TAPPED_UNDERLINE
             TrackableEvent.REDO_TAPPED -> Stat.EDITOR_TAPPED_REDO
             TrackableEvent.UNDO_TAPPED -> Stat.EDITOR_TAPPED_UNDO
+            TrackableEvent.REMOVE_AS_FEATURED_BUTTON_TAPPED -> Stat.EDITOR_TAPPED_REMOVE_FEATURED
+            TrackableEvent.SET_AS_FEATURED_BUTTON_TAPPED -> Stat.EDITOR_TAPPED_SET_FEATURED
```

Comment:
Should we remove the `SET_AS_FEATURED_BUTTON_TAPPED` case/event since we're not going to be implementing that "set via the Image block" feature in this PR?

While at it, not sure where `REMOVE_AS_FEATURED_BUTTON_TAPPED` is used 🤔. Should it be triggered somewhere?

Reply:
You're right that these weren't needed, I've gone ahead to remove them along with a few other functions that are no longer needed as part of this commit. The update to `libs/stories-android` referenced in the commits is also on my list to resolve after I've finished cleaning up on the commits on the `gutenberg-mobile` PR. 👍